### PR TITLE
Updates to Entries from Chaîne's Répertoire

### PR DESCRIPTION
**Pull request by smaugustine**
There are numerous other corrections and improvements to be made with regards to Chaîne's _Répertoire_, but these are simply a few particular cases which I have noted.

**Duplicate Entries**
- 153 & 216: [መልክአ፡ ፬እንስሳ፡](https://github.com/BetaMasaheft/Documentation/issues/1680), the difference being that 216 is the result of an extra stanza added at the beginning
- 2 & 267 & 284: መልክአ፡ ፋሲለደስ፡, 267 and 284 are the same text except ጸባኦት፡ has been replaced with ሥሉስ፡ ሕያው፡ in line 1, while 2 is the result of an apparent reordering of folios, such that stanza 40 begins on f. 101v and continues on f. 1v

**Titling**
Chaîne's _Répertoire_ is entirely unconcerned with titles, even when these were given by cataloguers or are known from (and consistent in) MSS themselves. I have replaced generic titles with their proper, traditional/conventional ones for 6, 125, 153, 175, 177, 190, 217, 218, 222, and 241. There is definitely more work to be done with titling, but in some cases this requires more careful investigation (I am thinking especially of those which have conventionally been numbered, often in printed books but sometimes in MSS, such as "መልክአ፡ ጊዮርጊስ፡ ሣልሳይ፡" and "መልክአ፡ ጊዮርጊስ፡ ራብዓይ፡," in which case it is not always clear precisely which is 3 or 4 or 1 etc. and extra work is needed to disambiguate them). The issue of titling is important because without it, MS contents appear as "1. Malkǝʾ-hymn to Mary 2. Malkǝʾ-hymn to Mary 3. Malkǝʾ-hymn to Mary ..." and become impossible to make sense of.

**Typos (from Chaîne)**
- a clear typo in 6 (ሥቃዱ፡ for ሥቃዩ፡)
- "ሰላም፡ መልክኦሙ፡" in 217, where the original MS (and the one other witness I am aware of) have "ሰላመ፡ መልክኦሙ፡"

**Bibliography**
- 218 (ለኵሎን፡ መልክእኪ፡) was one of the two hymns edited by Chrysostome Hayoz
- 284 (መልክአ፡ ፋሲለደስ፡) was, of course, included in Pereira's edition

### CODE REVIEW

**Comment by smaugustine** [comment]
As a footnote, some additional cases that will take quite a bit more work:

- LIT1860Malkea and LIT2835RepCh119, both for መልክአ፡ ሚካኤል፡ (ሰላም፡ ... መስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ።) which have been used in MS records 16 and 52 times respectively
- [መልክአ፡ ውዳሴ፡ (ማርያም፡)](https://github.com/BetaMasaheft/Documentation/issues/1538)

File: 2001-3000/LIT2894RepCh175.xml [hunk 1]
```diff
@@ -4,7 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Malkǝʾ-hymn to Our Lady Mary</title>
+           <title xml:lang="gez" xml:id="t1">መልክአ፡ ቍስቋም፡</title>
+           <title xml:lang="gez" corresp="#t1" type="normalized">Malkǝʾa Qʷǝsqʷām</title>
+           <title xml:lang="en" corresp="#t1">Image of Koskam</title>
```
Comment (eu-genia) [inline 2001-3000/LIT2894RepCh175.xml:9]: I believe we use Qusqam also as the English spelling

File: 2001-3000/LIT2722RepCh6.xml [hunk 2]
```diff
@@ -4,7 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Hymn to St George</title>
+           <title xml:lang="gez" xml:id="t1">መልክአ፡ ሥቃዩ፡ ለጊዮርጊስ፡</title>
+           <title xml:lang="gez" corresp="#t1" type="normalized">Malkǝʾa śǝqāyu laGiyorgis</title>
```
Comment (eu-genia) [inline 2001-3000/LIT2722RepCh6.xml:8]: la-Giyorgis

File: 2001-3000/LIT2937RepCh218.xml [hunk 3]
```diff
@@ -4,7 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Malkǝʾ-hymn to Our Lady Mary</title>
+           <title xml:lang="gez" xml:id="t1">ለኵሎን፡ መልክእኪ፡</title>
+           <title xml:lang="gez" corresp="#t1" type="normalized">Lakʷǝllon malkǝʾǝki</title>
```
Comment (eu-genia) [inline 2001-3000/LIT2937RepCh218.xml:8]: La-k

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you, the retitling is very helpful and important!

**Comment by smaugustine** [comment]
A further update for Chaîne no. 23, መልክአ፡ ልደታ፡.

The first two of Chaîne's witnesses, from Berlin, I can't speak to since the images aren't online, but the final witness, BL Add 19660 (f. 130v), I've updated in a new branch to reflect that it is a witness to no. 23 (and thus LIT5853HymnMary, which had been created for it, is a duplicate). Unnoticed by Wright was that the following folio (131r) is a separate hymn, Chaîne no. 221.

Further, it seems that no. 23, at least in the BL MS, is incomplete. EMIP 764 (f. 15r) contains a lengthier version of it. Of the 16 stanzas in EMIP 764, the text from BL Add 19660 matches stanzas 11-12.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**